### ISSUE TITLE
GetVolumePathName failed: Incorrect function.

### ISSUE THREAD

**Issue by audioscavenger** [darktable-org/darktable#10138]
**Describe the bug/issue**
when I open darktable with `-d imageio` I get 2 lines of error in the log:
```
(darktable.exe:23224): GLib-GIO-CRITICAL **: 14:15:46.006: GetVolumePathName failed: Incorrect function.
(darktable.exe:23224): GLib-GIO-CRITICAL **: 14:15:46.091: GetVolumePathName failed: Incorrect function.
```
I get them again when I close darktable. Not a blocker, I'm just curious about what is generating those.

**To Reproduce**
_Please provide detailed steps to reproduce the behaviour, for example:_
1. call darktable.exe -d imageio
2. open the log file `%LOCALAPPDATA%\Microsoft\Windows\INetCache\darktable\darktable-log.txt`

**Expected behavior**
no error message

**Platform**
* darktable version : 3.6.0.1
* OS                : Windows 10 x64 20h2
* Memory            : 32GB
* Graphics card     : GTX 1060
* Graphics driver   : 442.1
* OpenCL installed  : yes
* OpenCL activated  : yes

**Additional context**
sorry but I was not able to find the source code for the windows exe file or I would have looked at the line 23224 for you

**Comment by Mark-64** [comment]
I can't reproduce with the latest git master in Windows.
some Windows-specific bugs have been fixed in 3.6.1, I suggest you to install that and try again.
Please report the result

**Comment by Nilvus** [comment]
I put bug:fixed label as this is probably fixed in 3.6.1 (some paths error in Windows had been fixed in 3.6.1. You could download 3.6.1 here: https://github.com/darktable-org/darktable/releases/tag/release-3.6.1 (package at bottom page)
